### PR TITLE
Pi/referee-server

### PR DESCRIPTION
**Pull request by Quantumplation**
Adds a referee server to the hydra doom deployment.

Not sure if I did this right; not sure where to configure the image itself

### CODE REVIEW

File: crates/rpc/src/bin/metric_exporter/metrics.rs [hunk 1]
```diff
@@ -169,7 +163,8 @@ impl Metrics {
         *guard = Some(self.games_seconds.start_timer());
     }
 
-    pub fn end_game(&self) {
+    pub fn end_game(&self, remaining_players: i64) {
+        self.players_current.sub(remaining_players);
```
Comment (gonzalezzfelipe) [inline crates/rpc/src/bin/metric_exporter/metrics.rs:167]: Should this be `set`? given that we are talking about the remaining players
```suggestion
        self.players_current.sub(remaining_players);
```

Comment (Quantumplation) [reply crates/rpc/src/bin/metric_exporter/metrics.rs:167]: No; players current is supposed to be the current count of players; as a player joins, we increment it, and as a player disconnects we decrement it; but if a game ends, there might be remaining players that never disconnected so we subtract that amount.

In theory, of were doing one metric server per game, we could just do set(0) on startup, provided the stats aggregation will *sum* the amounts across each metrics server

File: playbook/doom-dev/main.tf [hunk 2]
```diff
@@ -52,6 +52,26 @@ variable "hydra_node_image" {
...
+variable "open_head_image" {
+  type        = string
+  description = "The Docker image to use for the open head component of the Hydra node."
+}
+
+variable "control_plane_image" {
+  type        = string
+  description = "The Docker image to use for the control plane component of the Hydra node."
+}
+
```
Comment (scarmuega) [inline playbook/doom-dev/main.tf:74]: You should only need the `dedicated_image` var. The other should all match the main image that we build out of this same repo.

Comment (Quantumplation) [reply playbook/doom-dev/main.tf:74]: oh i think this might be a merge snafu

**Review by scarmuega — APPROVED** [review]

**Pull request merged**